apps/content-analysis/package.json

```diff
@@ -70,8 +70,7 @@
 		"webpack-dev-server": "3.9.0",
 		"webpack-manifest-plugin": "2.2.0",
 		"workbox-webpack-plugin": "4.3.1",
-		"worker-loader": "^2.0.0",
-		"yoastseo": "^1.91.0"
```

Comment:
We removed the dependency on the old JavaScript monorepo, because that code conflicts with the changes to the `Paper` object.